ourstory.html

```diff
@@ -2,7 +2,8 @@
 <html>
 
 <head>
-    <title>PizzaParty</title>
+    <title>PizzaParty</title><link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta2/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-BmbxuPwQa2lc/FVzBcNJ7UAyJxM6wuqIj61tLrc4wSX0szH/Ev+nYRRuWlolflfl" crossorigin="anonymous">
```

Comment:
need to format this file. the link tag should be on its own line

Reply:
tag put on its own line
formatting done